keras/backend/common.py

```diff
@@ -13,13 +13,38 @@
 def epsilon():
     '''Returns the value of the fuzz
     factor used in numeric expressions.
+
+    # Return
+        Float (scalar)
+
+    # Example
+    ```python
+        >>> keras.backend.epsilon()
+        1e-07
+    ```
     '''
     return _EPSILON
 
 
 def set_epsilon(e):
     '''Sets the value of the fuzz
     factor used in numeric expressions.
+
+    # Argument
+        e: float. New value of epsilon
+
+    # Return
+        `None`
```

Comment:
When it's explicit, users can make sure that it doesn't return anything and that's what is intended - and it's not the case that docstring is omitted in documentation or a bug. Still do you think we should omit this?
([A Q&A in SO](http://stackoverflow.com/questions/27784179/docstrings-when-nothing-is-returned) mentioned it is not necessary to explicitly mention it or not according to [PEP257](https://www.python.org/dev/peps/pep-0257/). )